react-ui/src/index.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.scss';
-import Login from './components/Login/index';
+// import Login from './components/Login/index';
```

Comment:
the file is called "index.tsx" so that when you import things you only write `import Login from './components/Login';` and it uses "index" as default file

Reply:
this is just a theory remark